starter-code/css/main.css

```
@@ -6,3 +6,136 @@ Green: #00B172
 White: #FFF
 
 */
+
```

te faltó generar una clausula para eliminar todo el borde.
```suggestion
body{
  margin: 0;
}
```